README.md

```diff
@@ -6,33 +6,35 @@ What's new:
 - modern packages (biblatex, cleveref, better fonts)
 - less confusing directory structure
 - slightly more useful examples (figures, diagrams, tables, code listings),
-  structure hints, some goodies
-- autobuilding of abstract PDF/A files from metadata
+  structure&writing hints, some goodies
+- autobuilding of PDF/A abstracts directly from metadata
 - multiple variants of the front page
   - MFF with the new logo
   - "traditional" UK variant
   - Nature faculty & bioinformatics
-- Czech localization with nicely named references
-- Dockerized & CI build options
+- Czech localization with properly translated references
+- Automated docker-based & CI build options
 
-See the [pre-built version](build/thesis.pdf) for details
+See the [pre-built version](https://exaexa.github.io/better-mff-thesis/thesis.pdf) for details.
 
 ## CI configuration
 
 The repository contains valid configuration for both *GitLab* CI and the *GitHub* actions.
-No matter what GIT hosting you use, you can always download latest version of your thesis right from the artifacts!
+No matter what Git hosting you use, you can always download latest version of your thesis right from the artifacts!
+The GitHub version additionally pushes the files to GitHub pages to enabler easier sharing (incl. the link above); you can disable that by removing `.github/workflow/pages.yml`.
 
 ## How-to
 
-1. Type `make`, check that everything compiles. You should get a `thesis.pdf` that passes the PDF/A validation. If not, complain.
+1. Type `make`, check that everything compiles. You should get a `thesis.pdf` that passes the [PDF/A validation](https://github.com/mff-cuni-cz/cuni-thesis-validator). If not, complain.
```

Comment:
Since we already have the validator packed as Docker container and you started with the workflows, could we also have the validation as a step in the pipeline?